lib/cli/src/commands/create_exe.rs

```diff
@@ -961,22 +964,25 @@ fn get_module_infos(
     directory: &Path,
     atoms: &[(String, Vec<u8>)],
     object_format: ObjectFormat,
+    store: &Store,
 ) -> Result<BTreeMap<String, ModuleInfo>, anyhow::Error> {
     let mut entrypoint =
         get_entrypoint(directory).with_context(|| anyhow::anyhow!("get module infos"))?;
 
     let mut module_infos = BTreeMap::new();
     for (atom_name, atom_bytes) in atoms {
-        let store = Store::default();
-        let module = Module::from_binary(&store, atom_bytes.as_slice())
-            .map_err(|e| anyhow::anyhow!("could not deserialize module {atom_name}: {e}"))?;
+        let tunables = BaseTunables::for_target(store.engine().target());
```

Comment:
You don't need this

Reply:
Done in https://github.com/wasmerio/wasmer/pull/3483/commits/b26b48794cc7b00bb5f939dfdf40bba94e9ba86c